spec/UseAtomSpec.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+use Dxw\Iguana\Extras\UseAtom;
+
+describe('UseAtom', function () {
+	// initialisation
+	beforeEach(function () {
+		$this->useAtom = new \Dxw\Iguana\Extras\UseAtom();
+
+	});
+
+	afterEach(function () {
+		unset($this->useAtom);
+	});
+
+	describe("::register()", function () {
+		it('implement the registerable interface', function () {
+			expect($this->useAtom)->toBeAnInstanceOf(\Dxw\Iguana\Registerable::class, $this->useAtom);
+		});
+
+
+		it('should register actions correctly', function () {
+			allow('add_action')->toBeCalled();
+			expect('add_action')->toBeCalled()->times(2);
+			expect('add_action')->toBeCalled()->once()->with('init', [$this->useAtom, 'init']);
+			expect('add_action')->toBeCalled()->once()->with('wp_head', [$this->useAtom, 'wpHead']);
+
+			$this->useAtom->register();
+		});
+	});
+
+	describe('::init()', function () {
+		it('adds the default_feed filter and removes actions correctly', function () {
+			allow('add_filter')->toBeCalled();
+			allow('remove_action')->toBeCalled();
+
+
+			// Assertions
+			expect('add_filter')->toBeCalled()->once()->with('default_feed', [$this->useAtom, 'defaultFeed']);
+			expect('remove_action')->toBeCalled()->times(3);
+			expect('remove_action')->toBeCalled()->with('do_feed_rdf', 'do_feed_rdf', 10, 1);
+			expect('remove_action')->toBeCalled()->with('do_feed_rss', 'do_feed_rss', 10, 1);
+			expect('remove_action')->toBeCalled()->with('do_feed_rss2', 'do_feed_rss2', 10, 1);
+
+			$this->useAtom->init();
+
+
+		});
+
+
+
+		// Check that this code runs to completion without errors
```

Comment:
We can remove the check that the code runs to completion because we've used `expect()` here